support-docs/README.md

```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
+|tolerance|nominal tolerance of a capacitor|definitions.json#/unit| |
+|ratedVoltage|maximum voltage which may be applied continuously to a capacitance|definitions.json#/unit| |
+|dielectric|dielectric material used in the capacitor|String| |
+|polarized|describes whether the capacitor is polarized|Boolean| |
+|equivalentSerieResistance|equivalent series resistance (ESR) of the capacitor|definitions.json#/unit| |
+|temperatureCoefficient|change in capacitance when the temperature is changed|Number| |
+|minTemperature|minimum temperature under which a capacitor can be expected to reliably operate|definitions.json#/unit| |
+|maxTemperature|maximum temperature under which a capacitor can be expected to reliably operate|definitions.json#/unit| |
+|capacitorDerating|graph object to capture capacitance changes with voltage|definitions.json#/graphDefiniton| |
+|package|package size of capacitor|definitions.json#/package| |
+
+####  3.8.3	 Inductor
+
+|Property|Description|JSON Data Type|Required?|
+|:----|:----|:----|:----|
+|componentID|common component identifying information, such as mpn|definitions.json#/componentID|Yes|
+|pins|array of pin objects with associated properties|array of definitions.json#/pinSpec| |
+|value|inductor value|definitions.json#/unit|Yes|
+|tolerance|nominal tolerance of an inductor|definitions.json#/unit| |
+|ratedCurrent|maximum continuous current the inductor can handle|definitions.json#/unit| |
+|saturationCurrent|current where the inductor enters the magnetic state, and the inductance drops a specified amount|definitions.json#/unit| |
+|rmsCurrent|DC current that produces an inductor temperature rise of 40 degrees Celsius|definitions.json#/unit| |
+|selfResonantFrequency|frequency at which the inductor becomes a capacitor|definitions.json#/unit| |
+|dcResistance|DC resistance of the inductor|definitions.json#/unit| |
+|temperatureCoefficient|change in inductance when the temperature is changed|Number| |
+|minTemperature|minimum temperature under which a inductor can be expected to reliably operate|definitions.json#/unit| |
+|maxTemperature|maximum temperature under which a inductor can be expected to reliably operate|definitions.json#/unit| |
+|saturationCurve|graph object to capture inductor saturation with current|definitions.json#/graphDefiniton| |
+|resonantFrequencyCurve|graph object to capture inductor resonant frequency|definitions.json#/graphDefiniton| |
+|package|package size of inductor|definitions.json#/package| |
+
+####  3.8.4	 Common Mode Choke
+
+|Property|Description|JSON Data Type|Required?|
+|:----|:----|:----|:----|
+|componentID|common component identifying information, such as mpn|definitions.json#/componentID|Yes|
+|diffModeCutoff|frequency at which the differential mode attenuation equals -3dB|definitions.json#/unit| |
+|commonModeAttenuation|graph object to capture common mode attenuation of a common mode choke at various frequencies|definitions.json#/graphDefiniton| |

```

Comment:
"graph.json#/graphDefinition". It looks like we have to make these  fixes in the source files (passive.json, power.json,..)

Reply:
thanks for fixing this. I'll re-generate.